.moai/specs/SPEC-LSP-MULTI-006/progress.md

```
@@ -0,0 +1,66 @@
+|---------|----------|--------|--------|
+| `internal/lsp/config` | 100.0% | 85% | PASS |
+| `internal/lsp/subprocess` | 89.5% | 85% | PASS |
+| `internal/lsp/core` | 92.9% | 85% | PASS |
+| `internal/lsp/aggregator` | 96.8% | 85% | PASS |
+| `internal/cli` (lsp_doctor functions) | 85-100% | 85% | PASS |
+
+---
+
+## Files Changed
+
+### New Files
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)